src/Maths/Silk.NET.Maths/ClsCompliance.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System;
+
+[assembly: CLSCompliant(true)]
```

Comment:
Could this instead be defined using the csproj? It seems weird to have assembly attributes defined in a random file wheras the others are autogenerated using the csproj.

```xml
<ItemGroup>
    <AssemblyAttribute Include="System.CLSCompliant">
      <_Parameter1>true</_Parameter1>
    </AssemblyAttribute>
</ItemGroup>
```

Reply:
Didn't find how to do that, will apply!

Reply:
Well apparently
```cs
<ItemGroup>
    <AssemblyAttribute Include="System.CLSCompliant">
      <_Parameter1>true</_Parameter1>
    </AssemblyAttribute>
</ItemGroup>
```
will only function for `string`s, generating something like: `[assembly: System.CLSCompliant("true")]`
but I've renamed to AssemblyInfo.cs

Reply:
Huh that's a little annoying, all good.